### ISSUE TITLE
Cannot read property 'some' of undefined

### ISSUE THREAD

**Issue by dieguezguille** [react-spring/gltfjsx#32]
Receiving this message from the console when trying to run the `gltfjsx` command both locally and remotely. Seems it has to do with `jsdom`'s `whatwg-url` dependency. It also seems to work only when running version 1.0.0 of `gltfjsx`

`Cannot read property 'some' of undefined`

**Comment by giulioz** [comment]
```/private/tmp/gltfbug/node_modules/whatwg-url/dist/URL.js:84
  if (!globalNames.some(globalName => exposed.has(globalName))) {
                   ^

TypeError: Cannot read property 'some' of undefined
    at Object.exports.install (/private/tmp/gltfbug/node_modules/whatwg-url/dist/URL.js:84:20)
    at exports.installInterfaces (/private/tmp/gltfbug/node_modules/jsdom/lib/jsdom/living/interfaces.js:202:24)
    at setupWindow (/private/tmp/gltfbug/node_modules/jsdom/lib/jsdom/browser/Window.js:63:3)
    at new Window (/private/tmp/gltfbug/node_modules/jsdom/lib/jsdom/browser/Window.js:107:3)
    at exports.createWindow (/private/tmp/gltfbug/node_modules/jsdom/lib/jsdom/browser/Window.js:38:10)
    at new JSDOM (/private/tmp/gltfbug/node_modules/jsdom/lib/api.js:36:20)
    at globalJsdom (/private/tmp/gltfbug/node_modules/jsdom-global/index.js:29:18)
    at Object.<anonymous> (/private/tmp/gltfbug/node_modules/gltfjsx/gltfjsx.js:2:24)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)```

**Comment by giulioz** [comment]
This does not occur when running the app from a clone, only with `npx` or with `yarn run`

**Comment by giulioz** [comment]
![image](https://user-images.githubusercontent.com/1353142/91102911-7578e300-e66a-11ea-8497-3c65c4d5e9e8.png)

Investigating the lockfiles we can find that this package versions are different between the repo and the installed version.

**Comment by giulioz** [comment]
....and I confirm that fixing the lockfile to force the 8.0.0 versions fixes the bug.

Sorry for the investigation comments spam

**Comment by andrescrd** [comment]
It has not worked for me with a clone of the project. 
I removed the last parentheses of line 2 of the file "gltfjsx.js" and it started working

change this
require ('jsdom-global') ()

for this
require ('jsdom-global')

**Comment by giulioz** [comment]
> It has not worked for me with a clone of the project. 
> 
> I removed the last parentheses of line 2 of the file "gltfjsx.js" and it started working
> 
> 
> 
> change this
> 
> require ('jsdom-global') ()
> 
> 
> 
> for this
> 
> require ('jsdom-global')

Actually, you're supposed to call the require as function to load the jsdom environment

**Comment by andrescrd** [comment]
I was using npm. I switched to yarn and it works perfectly. Thank you :D

**Comment by gsimone** [comment]
This is now fixed in version 2.0.5 for people using npx to run the command 👍
